README.md

```diff
@@ -210,6 +210,46 @@ fastify.get('/', (req, rep) => {
 })
 ```
 
+### Other cases of manual signing / unsigning cookies
+
+Sometimes the service under test should only accept requests with signed cookies, but it does not generate them itself.
+
+**Example:**
+
+```js
+import { signerFactory } from "@fastify/cookie";
+
+const signerFactory = require('@fastify/cookie/signer');
```

Comment:
This is a duplicate declaration. We also should not recommend the require form shown in the second declaration.